en/drafts/originals/designing-a-timeline-tabletop-simulator.md

```diff
@@ -64,354 +60,352 @@ In 2013, game designer Eric Zimmerman published a “Manifesto for a Ludic Centu
 
-{% include figure.html filename="or-en-designing-a-timeline-tabletop-simulator-7.png" alt="Two pages of cards in a grid format. The left hand page of cards feature years printed in bold and the right hand side of cards are the same but without the year printed." caption="Figure 7: The simplest way to have _nanDECK_ generate a card side without the date being printed is to duplicate the information used to make cards without text in the date field." %}
+{% include figure.html filename="or-en-designing-a-timeline-tabletop-simulator-7.png" alt="Two pages of cards in a grid format. The left hand page of cards feature years printed in bold and the right hand side of cards are the same but without the year printed." caption="Figure 7. The simplest way for nanDECK to generate a card side without the date is to duplicate the information and remove the 'year' fied from one half." %}
 
 This is admittedly not the most efficient method, as the information needed to generate new cards must be added twice into the spreadsheet (one entry with the year and one entry without), but it does produce the desired end result.
 
 ### Lines 11 and 12: DUPLEX and PRINT
 
-One of the greatest strengths of _nanDECK_ is its ability to generate synchronized backs and faces of cards. It does this through the DUPLEX and PRINT directives.
+One of the greatest strengths of nanDECK is its ability to generate synchronized faces and backs of cards. It does this through the `DUPLEX` and `PRINT` directives.
 
-> DUPLEX
-> This directive copies a card (or a range of cards) to another position (or range) calculated automatically by the software, it is useful to manage duplicates or synchronize the front face and back of cards for a duplex printing
+> DUPLEX      
+> This directive copies a card (or a range of cards) to another position (or range of positions) calculated automatically by the software. It is useful to manage duplicates, or synchronize the face and back of cards for a duplex printing.
 
-> DUPLEX = “range front”, “range back”, number
+```
+DUPLEX = “range front”, “range back”, number
+```
 
-In order to align the cards with years printed on them, it is only necessary to designate their range. In the case of my personal deck, there are 59 unique cards, with the fronts being generated from range 1-59 in my spreadsheet and the backs generated from range, 60-118.
+In order to align the card fronts and backs, designate the range you want to duplicate. In the case of my personal deck, there are 59 unique cards: the fronts will be generated from cards in the 1-59 range, and the backs from the cards in the 60-118 range.
```

Comment:
I'm not sure I understand exactly why we are counting cards to 59 (and thus doubling to 118) here, as I believe the deck only contains 58. Perhaps this is a small error?

Reply:
I checked my code and it's not an error. I believe that nanDECK counts oddly because it knows to skip the first row as it contains field names.

Reply:
If that's the case, perhaps we should add a sentence to that effect, as it's not very clear to me at the moment. Here you say 'In the case of my personal deck, there are 59 unique cards' – but this doesn't seem right! Am I correct in understanding that there are 58 unique cards and images, but the card #58 would be on row 59 of the spreadsheet? 

Line 330 says 'there are 58 images, so {(IMAGES)} returns `58`. Knowing this, you'll see that at line 6 of Figure 5 below, `IMAGE="1-{(IMAGES)}"` instructs nanDECK to print the images from rows 1 through 58.'

 Line 367 also says 'In the spreadsheet, the first 58 rows provide an image, a description, and a year, to be printed on the face of the card. The next 58 rows only provide an image and a description, because the 'year' field is left empty.' 
 
Perhaps you'll be able to notice if there is something we should change to make it clear/consistent? Thank you for your patience with this!

Reply:
I can indeed see from the spreadsheet in Figure 7 that the final image with a 'year' is at row 60...

Reply:
Thank for you for your persistence with this because you are correct. The deck is 59 cards. My apologies - I should have caught that on the first pass.